Source/Tools/CodeSizeValidation/Main.cpp

```diff
@@ -447,7 +447,18 @@ class TestEnvLoader final : public FEXCore::Config::Layer {
 
     for (auto& it : EnvConfigLookup) {
       if (auto Value = GetVar(it.first); Value) {
-        Set(it.second, *Value);
+#define OPT_BASE(type, group, enum, json, default) // Nothing
+#define OPT_STRARRAY(group, enum, json, default)                   \
+  if (it.second == FEXCore::Config::ConfigOption::CONFIG_##enum) { \
+    AppendStrArrayValue(it.second, *Value);                        \
+  }
+
+        if (false) {
```

Comment:
what's with the `if (false)` here?

Reply:
Oops, the define should have used an `else if` so the `if (false)` is the start of the chain of generated code hitting `OPT_STRARRAY` and everything else falls down the final `else` path.

Fixed the define.